docusaurus/docs/shared/prerequisites.md

```diff
@@ -0,0 +1,6 @@
+- Knowledge about how data sources are implemented in the frontend.
+- Grafana 7.0
+- Go ([Version](https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/templates/backend/go.mod#L3))
```

Comment:
Again, Go/Mage is only required for backend plugins. We could create a separate prerequisites-backend.md section for these two perhaps?